.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1669,6 +1671,19 @@ public class FileClient {
         public static bool operator !=(FileUploadPurpose left, FileUploadPurpose right);
         public override readonly string ToString();
     }
+    public class IncrementalUpload : ClientResult {
+        public ContinuationToken ContinuationToken { get; }
+        public DateTimeOffset CreatedAt { get; }
+        public DateTimeOffset ExpiresAt { get; }
+        public string Id { get; }
+        public virtual ClientResult AddDataPart(string jobId, BinaryContent content, string contentType, RequestOptions options = null);
```

Comment:
I was discussing this with anne today. we thought we should remove parameters from protocol methods if the parameter is stored in the subclient.